cli/provider/docker.go

```diff
@@ -125,13 +125,20 @@ func RunInDocker(ctx context.Context, logger *zap.Logger) error {
 		if ctx.Err() == context.Canceled {
 			return ctx.Err()
 		}
+		if exitError, ok := err.(*exec.ExitError); ok {
+			exitCode := exitError.ExitCode()
+			if exitCode == 143 {
+				logger.Info("Command terminated with exit code 143 (SIGTERM), treating as successful exit")
+				return nil
+			}
+		}
```

Comment:
I think we now give exit code 1 if the tests cases fail we need to handle this properly around keploy binary and keploy docker communication.

Reply:
Making another condition to send a debug log with exit code 1